### ISSUE TITLE
Barrier Mouse Movement Issue on Windows 10 High-DPI Display with 125% Scaling

### ISSUE THREAD

**Issue by Echo23Bravo** [debauchee/barrier#1918]
### What happened?

I am using Barrier on two computers: a Linux Mint machine (version 2.3.4) as the server and a Windows 10 client (version 2.4.0) with two monitors. The Windows 10 machine has a center monitor running at 2048x1152 resolution and a right monitor at 1920x1080 resolution. When I change the scaling on the right monitor to 125%, the mouse won’t move to the bottom and right of the screen. It’s as if there is an invisible barrier to mouse movement.  The area that can't be accessed seems to be a ratio of the scaling, i.e. the part of the screen that is inaccessible is 25% bigger than the accessible portion.

I expected that changing the scaling on the right monitor would not affect mouse movement and that I would still be able to move the mouse to all areas of the screen.

### Version

v2.4.0

### If applicable, where did you install Barrier from?

Win 10: from github release.
Linux Mint: Flatpak, Flathub, Stable

### What OSes are you seeing the problem on? (Check all that apply)

Linux, Windows

### What OS versions are you using?

Win 10 Version 10.0.19044 Build 19044
Linux Mint 20.3
Cinnamon 5.2.7
Kernel 5.4.0-88-generic

### Relevant log output

```shell
Log shows nothing on debug level
```

### Any other information

_Pun not intended_

**Comment by dogasantos** [comment]
plus one

**Comment by mlester82** [comment]
same problem. current work around is to install barrier 2.3.4 on Windows as barrier server, mouse no longer invisible on linux

**Comment by Zijie-Tian** [comment]
plus one

**Comment by mezentsevw** [comment]
> same problem. current work around is to install barrier 2.3.4 on Windows as barrier server, mouse no longer invisible on linux


THANK YOU SO MUCH, for a long time I could not understand what was the matter and thanks to you I solved this problem

**Comment by FurmanSK** [comment]
Yeah plus one. Got 4k monitor and did 150% DPI and mouse doesn't work on my mac. Going back to 2.3.4 didn't work for mac client.

This comment fixed my issue. [1506](https://github.com/debauchee/barrier/pull/1506#issuecomment-1013758495)

**Comment by igordovgaluk** [comment]
the problem is fixed by https://github.com/debauchee/barrier/pull/1506
download v2.4.0+ build (non-release) from CI/CD

**Comment by danielponte** [comment]
> the problem is fixed by #1506 download v2.4.0+ build (non-release) from CI/CD

How do I download a non-release version for Windows?

**Comment by midenok** [comment]
> the problem is fixed by [#1506](https://github.com/debauchee/barrier/pull/1506) download v2.4.0+ build (non-release) from CI/CD

@igordovgaluk Where are those nightly builds? The link would be nice.

**Comment by nbolton** [comment]
Please try Deskflow or Input Leap as Barrier is no longer in development.

https://github.com/deskflow/deskflow
https://github.com/input-leap/input-leap

Both are compatible with Barrier client/server so you can try on one of your computers.

If this is still an issue in those projects, we would appreciate a cross-post of this issue.

Not sure which to use? The 'Insights' tab of each project may help you decide.

**Comment by midenok** [comment]
2.3.4 worked for me. As everything works I'm happy with it and I don't want to spend anymore minutes on this. While you proposing some stupid investigation and comparison again, I pass. Nobody updated this project status btw and added links to the newer project.
